src/pages/Home/index.tsx

```diff
@@ -71,6 +75,8 @@ const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
       <img style={{ width: '24rem' }} src={imageLink} alt={name}></img>
       <p>{description}</p>
       <BobaModalForm
+        milkList={milkList}
+        toppingsList={toppingsList}
```

Comment:
change and use props from sampleAPI

Reply:
we discussed this , i am not passing props , i am passing the whole list of Milk and Topping - pretty sure u said , it is ok then, ( u might of though i am passing props but i am passing array of obejcts of milk and topping